src/components/tooltip/ToolTip.stories.tsx

```diff
@@ -9,105 +8,145 @@
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
 
-import { storiesOf } from "@storybook/react";
-import { ToolTip, ToolTipDirection, ToolTipSize } from "@dellstorage/clarity-react/forms/tooltip/ToolTip";
+import {storiesOf} from "@storybook/react";
+import {ToolTip, ToolTipDirection, ToolTipSize} from "@dellstorage/clarity-react/forms/tooltip/ToolTip";
 import "styles/components/ToolTip.scss";
 
 storiesOf("ToolTip", module)
     .add("Tooltip sizes", () => (
-        <div style={{ paddingTop: "75px" }}>
-            <div style={{ float: "left" }}>
-                <ToolTip size={ToolTipSize.EXTRA_SMALL}>Lorem</ToolTip>
+        <div
+            style={{
+                display: "flex",
+                justifyContent: "space-around",
+                alignItems: "center",
+                height: "80vh",
+                width: "75vw",
+            }}
+        >
+            <div>
+                <ToolTip size={ToolTipSize.EXTRA_SMALL} isSolidIcon>
+                    Content size: 16px
+                </ToolTip>
                 Extra Small
             </div>
-            <div style={{ float: "left", marginLeft: "20px" }}>
-                <ToolTip size={ToolTipSize.SMALL}>Lorem ipsum sit</ToolTip>
+            <div style={{marginLeft: "20px"}}>
+                <ToolTip size={ToolTipSize.SMALL} isSolidIcon iconSize={20}>
+                    Content size: 20px
+                </ToolTip>
                 Small
             </div>
-            <div style={{ float: "left", marginLeft: "20px" }}>
-                <ToolTip size={ToolTipSize.MEDIUM}>Loren ipsum dolor sit amet, ipsum</ToolTip>
+            <div style={{marginLeft: "20px"}}>
+                <ToolTip size={ToolTipSize.MEDIUM} isSolidIcon iconSize={24}>
+                    Content size: 24px
+                </ToolTip>
                 Medium
             </div>
-            <div style={{ float: "left", marginLeft: "20px" }}>
-                <ToolTip size={ToolTipSize.LARGE}>Loren ipsum dolor sit amet, consectetur adipisicing elit</ToolTip>
+            <div style={{marginLeft: "20px"}}>
+                <ToolTip size={ToolTipSize.LARGE} isSolidIcon iconSize={30}>
+                    Content size: 30px
+                </ToolTip>
                 Large
             </div>
         </div>
     ))
     .add("Tooltip directions", () => (
-        <div style={{ paddingTop: "75px" }}>
-            <div style={{ float: "left" }}>
-                <ToolTip size={ToolTipSize.MEDIUM} direction={ToolTipDirection.TOP}>
-                    Lorem
+        <div
+            style={{
+                display: "flex",
+                justifyContent: "space-around",
+                alignItems: "center",
+                height: "80vh",
+                width: "75vw",
+            }}
```

Comment:
Please create constant for this style as it is repeating

Reply:
Done